### PR TITLE
feat: add tailwind beta release

### PR DESCRIPTION
**Pull request by mcwinter07**
## Why
We're splitting tailwind from the design-tokens package to ensure that versioning this will likely undergo as we continue to develop in its early stages does not impact our consumers of design tokens that are not using tailwind.

## What
- will release @kaizen/tailwind as 0.0.1
- will add kaizen presets, story tables and documentation on local development

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// ts-check
+/** @type {import('tailwindcss').Config} */
+
+const kzPresets = require("@kaizen/tailwind").TailwindPreset
```
Comment (gyfchong) [inline tailwind.config.js:4]: I wonder if naming this `TailwindPreset` is now redundant? Thoughts on calling it `Preset`?

Comment (mcwinter07) [reply tailwind.config.js:4]: True, now it's in its own package probs doesn't need to be explicitly called out.

File: packages/tailwind/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Install
```
Comment (gyfchong) [inline packages/tailwind/README.md:1]: Should include examples of tailwind config and VSCode recommendations

Comment (gyfchong) [reply packages/tailwind/README.md:1]: And a huge disclaimer that API is subject to change lol

Comment (mcwinter07) [reply packages/tailwind/README.md:1]: True that, there's some guidance in the FE Template I should link back too as well

**Review by JakePitman — APPROVED** [review]

**Review by gyfchong — APPROVED** [review]

**Pull request merged**